test/util.dart

```diff
@@ -2,7 +2,7 @@
 library react.test.util;
 
 import 'package:js/js.dart';
-import 'package:react/react_client/js_interop_helpers.dart';
+import 'package:js/js_util.dart';
```

Comment:
so package:js/js_util.dart actually just exports `dart:js_util` so id say using it is better than depending on an external library. but you could also just keep using js_interop_helpers.dart until its fully removed too. your call.
```suggestion
import 'dart:js_util';
```